assertk/src/commonMain/kotlin/assertk/failure.kt

```diff
@@ -137,6 +142,12 @@ internal class SoftFailure(
     }
 }
 
+private class GroupedFailuresError(heading: String?, failures: List<Throwable>) : AssertionError() {
+    private val multipleFailuresError = MultipleFailuresError(heading, failures)
+
+    override val message: String? get() = multipleFailuresError.message
+}
```

Comment:
I could inherit from `MultipleFailuresError` here, but that would require complicating `if` in the line `110`, so I believe this is cleaner.